docs/compose-file.md

```diff
@@ -623,7 +623,7 @@ An example:
 
 #### link_local_ips
 
-> [Version 2.1 file format](#version-2.1) only.
+> [Added in version 2.1](#version-2.1).
```

Comment:
Sorry, this probably needs to be "file format version 2.1" or something like that, so it's not confused with the Compose version, right?